### PR TITLE
pmd-changes

### PR DESCRIPTION
**Pull request by tanvi029**

### CODE REVIEW

**Comment by pranavrd** [comment]
ok to test

File: google_checks.xml [hunk 1]
```diff
@@ -25,12 +26,12 @@
...
     <module name="BeforeExecutionExclusionFileFilter">
-        <property name="fileNamePattern" value="module\-info\.java$"/>
+        <!--<property name="fileNamePattern" value="module\-info\.java$,target"/>-->
+        <property name="fileNamePattern" value="module\-info\.java$,target$,AsyncServiceVertxEBProxy\.java$,^(?!.*(VertxEBProxy\.java|VertxProxyHandler\.java|[\\/]VertxEBProxy\.java|[\\/]VertxProxyHandler\.java)).*$"/>
     </module>
     <!-- https://checkstyle.org/config_filters.html#SuppressionFilter -->
     <module name="SuppressionFilter">
-        <property name="file" value="${org.checkstyle.google.suppressionfilter.config}"
-                  default="checkstyle-suppressions.xml" />
+        <property name="file" value="$AsyncServiceVertxEBProxy.java$"/>
```
Comment (pranavrd) [inline google_checks.xml:34]: Same as above

**Review by pranavrd — CHANGES_REQUESTED** [review]
Resolve above comments

**Review by pranavrd — APPROVED** [review]
LGTM

File: src/main/java/iudx/catalogue/server/apiserver/MlayerApis.java [hunk 2]
```diff
@@ -48,11 +46,11 @@ public void setHost(String host) {
...
 
-  private Api api;
+  // private Api api;
 
-  public MlayerApis(Api api) {
-    this.api = api;
-  }
+  // public MlayerApis(Api api) {
+  //  this.api = api;
+  //}
```
Comment (kailash) [inline src/main/java/iudx/catalogue/server/apiserver/MlayerApis.java:54]: remove commented code

**Review by kailash — CHANGES_REQUESTED** [review]

**Review by kailash — APPROVED** [review]

**Pull request merged**